frontend/src/Components/Functional Body/FunctionalBody.tsx

```diff
@@ -0,0 +1,136 @@
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+      species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/755871/pexels-photo-755871.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+      species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/2829336/pexels-photo-2829336.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/6034584/pexels-photo-6034584.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/869258/pexels-photo-869258.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/755871/pexels-photo-755871.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/2829336/pexels-photo-2829336.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/6034584/pexels-photo-6034584.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/869258/pexels-photo-869258.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+    {
+      img: "https://images.pexels.com/photos/755871/pexels-photo-755871.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260",
+      title: "Name",
+      text: `Lizards are a widespread group of squamate reptiles, with over 6,000
+        species, ranging across all continents except Antarctica`,
+    },
+  ]);
```

Comment:
Move the initial value of the state to an external file and import it here, it will make your code more readable